Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1673,6 +1673,18 @@ def lhegensim(fragment,howMuch):
                                        }
     if cust!=None : upgradeStepDict['GenSimHLBeamSpotFull'][k]['--customise']=cust
     if era is not None: upgradeStepDict['GenSimHLBeamSpotFull'][k]['--era']=era
+
+    upgradeStepDict['GenSimHLBeamSpotFull14'][k]= {'-s' : 'GEN,SIM',
+                                       '-n' : 10,
+                                       '--conditions' : gt,
+                                       '--beamspot' : 'HLLHC14TeV',
+                                       '--datatier' : 'GEN-SIM',
+                                       '--eventcontent': 'FEVTDEBUG',
+                                       '--geometry' : geom
+                                       }
+    
+    if cust!=None : upgradeStepDict['GenSimHLBeamSpotFull14'][k]['--customise']=cust
+    if era is not None: upgradeStepDict['GenSimHLBeamSpotFull14'][k]['--era']=era
```

Comment:
@davidlange6 This proposes two beamspots, both are HL-LHC beamspots but consistent with the COM energies of the samples (13 and 14 TeV, respectively). The distinction is made here since there are differences in the beamspot depending on the beam kinematics.